### PR TITLE
fix(PROVES-169): Resolve issue where change log entries for list items were showing blank

### PR DESCRIPTION
**Pull request by kehh**
Backport fix from https://github.com/collectiveaccess/providence/commit/ca19fff1e4153d56c594e768b955c3d8b7b27b93

### CODE REVIEW

**Comment by kehh** [comment]
Doesn't fix the problem, just fixes the symptom.

File: app/lib/ApplicationChangeLog.php [hunk 1]
```diff
@@ -1361,8 +1360,10 @@ static private function makeChangeLog($data, $options=null) {
 								// Convert list-based attributes to text
 								if ($vn_list_id = $t_element->get('list_id')) {
 									$t_list = new ca_lists();
-									$vs_attr_val = $t_list->getItemFromListForDisplayByItemID($vn_list_id, $vs_attr_val, true);
+									dump($vn_list_id);
```
Comment (bri-gaia) [inline app/lib/ApplicationChangeLog.php]: @kehh  remove dumps?

Comment (kehh) [reply app/lib/ApplicationChangeLog.php]: I just force pushed that change

**Comment by bri-gaia** [comment]
LGTM

**Pull request merged**